### DISCUSSION TITLE
Als Schüler will ich mein Mobilitätsfahrzeug auswählen zur Schule berechnet haben, damit ich planen kann wie ich zur Schule komme und wann ich los muss.

### DISCUSSION THREAD

**Discussion by Kronprinz03** [SE-TINF22B2/G2-Schoolify#18] [User Stories]
### Beschreibung:
Um die API Anbindung auf die jeweilige Person abzustimmen, sollte es die Möglichkeit geben bei der Berechnung ein Mobilitätsfahrzeug (Fahrrad, Bahn, zu Fuß oder Auto) festzulegen. Damit sollte dann automatisch die Dauer und der Weg angezeigt werden, z.B. auf einer Karte.

**Risiken**: Api akzeptiert keine eingaben von Fahrzeugen

### Akzeptanz-Kriterien:
- [ ] Schüler kann selbst auswählen wie er zur Schule kommen will 
- [ ] Dauer und Weg werden daraufhin angezeigt

**Comment by laralngn** [comment]
Entweder würde ich die Dauer aus dem Titel entfernen (und dann aus der Beschreibung) oder so lassen wie ich es gerade angepasst habe (mit Dauer in der Beschreibung ergänzt).

**Comment by two-Gee** [comment]
nicht mehr in requirements
